.github/CONTRIBUTING.md

```diff
@@ -96,7 +97,21 @@ If you are adding new assets that are not already explicitly licensed under CC-B
 
 If at any point you are confused or unsure of an asset's license or our policy, ask a Maintainer to help you.
 
-## 7. Banned Content
+## 7. Pull Request Standards
+
+You should complete the pull-request template in its entirety.
+
+Any pull-request that does not adequately complete the provided template may be closed or marked 'do not merge' by maintainers.
+ - Balance changes must indicate that they affect the balance of the game.
```

Comment:
The wording here is a bit awkward. I'd maybe suggest something like this

```suggestion
 - If you make any changes that affect the balance of the game, they must be mentioned as balance changes. Note that this also applies if you're fixing something, even if the result is what was originally intended.
```

Reply:
It might be poorly phrased, but I feel like we've had cases where significant balance changes were PR'd under the banner of fixes, because the original code's intent seemed to be different than what it actually did. What I mean is, even if the balance changes are the result of a fix, the rule should apply in the same way, treating it as a balance change based on what the difference is.